# Competition configuration options
options:
  insert_models_over_bins: true
  insert_models_over_shelves: true
  belt_population_cycles: 0

models_over_bins:
   bin1:
     models:
       apple_red:
         xyz_start: [0.2, 0.2, 0.0]
         xyz_end: [0.4, 0.4, 0.0]
         rpy: [0, 0, 0]
         num_models_x: 2
         num_models_y: 2
   bin2:
     models:
       apple_green:
         xyz_start: [0.1, 0.2, 0.0]
         xyz_end: [0.4, 0.5, 0.0]
         rpy: [0, 0, 'pi/4']
         num_models_x: 2
         num_models_y: 3

time_limit: -1
